.github/mergify.yml

```diff
@@ -5,7 +5,7 @@ queue_rules:
 pull_request_rules:
   - name: automatic merge for Scala Steward
     conditions:
-      - author=tna-digital-archiving-jenkins
+      - author=tna-tna-da-bot
```

Comment:
I see two times `tna` hope it's not a typo error.

Reply:
Thanks for spotting that, cut and paste error